<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="bpmBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>70</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>110</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>120</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>130</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>140</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>160</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>170</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>180</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Repeat</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="repeatBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Samplerate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="samplerateBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>8000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22050</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>44100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>48000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>88200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>96000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>176400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>192000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>352800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>384000</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Synth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="synthBox">
     <item>
      <property name="text">
       <string>square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bell/click</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Emphasing</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="emphasingBox">
     <item>
      <property name="text">
       <string>normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>no emphasizing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>emphasized only</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="toolTip">
      <string>Copy</string>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Command:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
